plugins/module_utils/gitlab.py

```diff
@@ -21,15 +21,30 @@
 
 import traceback
 
+
+def _determine_list_all_kwargs(version):
+    gitlab_version = LooseVersion(version)
+    if gitlab_version >= LooseVersion('4.0.0'):
+        # 4.0.0 removed 'as_list'
+        return {'iterator': True, 'per_page': 100}
+    elif gitlab_version >= LooseVersion('3.7.0'):
+        # 3.7.0 added 'get_all'
```

Comment:
current version of gitlab is 16.x and even the history page lists 7.14 as oldest version (from 2015)  
https://about.gitlab.com/releases/#historic-releases  
Can we drop the version handling?

Reply:
This version check isn't for the version of the GitLab instance, but the version of the [python-gitlab module](https://python-gitlab.readthedocs.io/en/stable/)